connect/runtime/src/main/java/org/apache/kafka/connect/util/TopicAdmin.java

```diff
@@ -233,7 +234,7 @@ public boolean createTopic(NewTopic topic) {
                     log.debug("Found existing topic '{}' on the brokers at {}", topic, bootstrapServers);
                     continue;
                 }
-                if (cause instanceof UnsupportedVersionException) {
+                if (cause instanceof UnsupportedVersionException || cause instanceof ClusterAuthorizationException) {
```

Comment:
It seems like it would be useful to know in the log message below which case has actually happened. Perhaps we should use a separate `if` branch with a separate log message for this case?

Reply:
Very well. I wanted to keep the change minimal, but the suggestion makes sense.